html/changelogs/alberyk-steelunathi.yml

```diff
@@ -0,0 +1,6 @@
+author: Alberyk, Kyres1
+
+delete-after: True
+
+changes: 
+  - rscadd: "Added a the Aut'akh unathi subspecies."
```

Comment:
Added the.
Also, more details could be cool.

Reply:
All changes besides adding the species are not really noticiable by the players, and their mechanics and differences should be on the wiki.

Reply:
that's valid